src/filing/actions/fetchNewFiling.js

```diff
@@ -15,7 +16,10 @@ export default function fetchNewFiling(filing) {
             dispatch(receiveError(json))
             throw new Error(json && `${json.status}: ${json.statusText}`)
           }
-          return dispatch(receiveFiling(json))
+          if (!hasError) {
+            dispatch(receiveFiling(json))
+            return dispatch(receiveLatestSubmission({ lei: filing.lei }))
```

Comment:
I was creating a new submission after the creation of a new filing.

After discussing with Bhaarat, that would not accurately reflect the user's actions and also mess up some analytics. 

I'm currently unable to test my latest change because `dev-default` is unreachable, but I will now be adding a dummy submission to the `store` after creating a new filing.   I think this will allow the Institutions page to properly display the user's current filing progress without creating anything on the backend.

Reply:
Ahh, yeah thanks for taking that out, I noticed that when creating my other branch but forgot to bring it up once I abandoned that approach